Tasks/Common/find-files-legacy/findfiles.legacy.ts

```diff
@@ -44,7 +44,7 @@ function extractPattern(patterns: string[], cwd: string): any {
     let excludePatterns: string[] = [];
 
     patterns.forEach(p => {
-        p.split(',').forEach(inputFilter => {
+        p.split(';').forEach(inputFilter => {
```

Comment:
still I think it would be better to override this by passing a seperator in the function parameters. Though its not a blocker, if you feel its too much overhead then ignore.